### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by rijkerd**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -23,11 +23,22 @@
 
 /* dependencies */
```
Comment (lykmapipo) [inline src/index.js]: Remove this comment

File: src/index.js [hunk 2]
```diff
@@ -40,8 +51,40 @@ const info = pkg(
   'contributors'
 );
 
-/* export router api version */
-const apiVersion = router.version;
+/**
+ * @name Jurisdiction
+ * @description Predefine model
```
Comment (lykmapipo) [inline src/index.js]: Rename to Jurisdiction model

File: test/integration/http.spec.js [hunk 3]
```diff
@@ -1,150 +0,0 @@
-/* dependencies */
```
Comment (lykmapipo) [inline test/integration/http.spec.js]: Remove this comment

File: test/integration/jurisdiction.get.spec.js [hunk 4]
```diff
@@ -1,128 +1,120 @@
 /* dependencies */
```
Comment (lykmapipo) [inline test/integration/jurisdiction.get.spec.js]: Remove this comment

File: test/integration/jurisdiction.getById.spec.js [hunk 5]
```diff
@@ -1,86 +1,78 @@
 /* dependencies */
```
Comment (lykmapipo) [inline test/integration/jurisdiction.getById.spec.js]: Remove this comment

File: test/integration/jurisdiction.http.spec.js [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+        expect(body.total).to.exist;
+        expect(body.limit).to.exist;
+        expect(body.skip).to.exist;
+        expect(body.page).to.exist;
+        expect(body.pages).to.exist;
+        expect(body.lastModified).to.exist;
+        done(error, body);
+      });
+  });
+
```
Comment (lykmapipo) [inline test/integration/jurisdiction.http.spec.js]: Add schema spec. [Check](https://github.com/lykmapipo/predefine/blob/master/test/integration/predefine.http.spec.js#L57)

File: test/integration/jurisdiction.http.spec.js [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  it('should handle HTTP GET on /jurisdictions/id:', done => {
```
Comment (lykmapipo) [inline test/integration/jurisdiction.http.spec.js]: Add export spec. [Check](https://github.com/lykmapipo/predefine/blob/master/test/integration/predefine.http.spec.js#L62)

File: test/integration/http.spec.js (hunk 3 above)
Comment (rijkerd) [reply test/integration/http.spec.js]: Renamed this file to jurisdiction.http.spec.js

**Pull request merged**